src/Psd/PsdCompilerMacros.h

```diff
@@ -144,3 +144,11 @@
 	#define static_assert(condition, message)			typedef char PSD_JOIN(static_assert_impl_, __LINE__)[(condition) ? 1 : -1]

```

Comment:
Looks like this needs to be revisited.
When defining PSD_STATIC_ASSERT to static_assert((condition)) down below, this function macro would no longer work, since it expects two arguments.

Reply:
Surely.
```
#if __cpp_static_assert >= 201411
	#define PSD_STATIC_ASSERT(condition)				static_assert((condition))
#else
	#define PSD_STATIC_ASSERT(condition)				static_assert((condition), "")
#endif
```
should be replaced to
```
#define PSD_STATIC_ASSERT(condition)				static_assert((condition), "")
```

but, error messages may be inappropriate.